src/transforms/tag_cardinality_limit/config.rs

```diff
@@ -86,7 +86,7 @@ const fn default_value_limit() -> usize {
 }
 
 pub(crate) const fn default_cache_size() -> usize {
-    5000 * 1024 // 5KB
+    5 * 1024 // 5KB
```

Comment:
This is a potential behavior change, in that it will cause more false positives for sites that have not explictly configured the cache size.